syntax = "proto3";

message TradeRequest {
    string stockName = 1;
    int64 itemSize = 2;
    string type = 3;
}